01_Data/src/layers/sequelize/model/Authorization/AdditionalInfo.ts

```diff
@@ -3,17 +3,17 @@
 //
 // SPDX-License-Identifier: Apache 2.0
 
-import { OCPP2_0_1_Namespace, OCPP2_0_1 } from '@citrineos/base';
+import {OCPP2_0_1_Namespace, OCPP2_0_1, Namespace} from '@citrineos/base';
 import { BelongsToMany, Column, DataType, Model, Table } from 'sequelize-typescript';
 import { IdToken } from './IdToken';
 import { IdTokenAdditionalInfo } from './IdTokenAdditionalInfo';
 
 @Table
-export class AdditionalInfo extends Model implements OCPP2_0_1.AdditionalInfoType {
```

Comment:
`AdditionalInfo` is a concept only in OCPP 2.0.1. I think we don't need to generalize it.